### PR TITLE
GOTTA GO FAST

### PR DESCRIPTION
**Pull request by DamianX**
This makes world start (measured from start to "initializations complete") 2 minutes and ~~30~~ 40 seconds faster on my machine.
https://www.youtube.com/watch?v=_Rp5syLd1LU

Thanks to @clusterfack for showing me how to do this.

### CODE REVIEW

**Comment by Probe1** [comment]
It boots and I vote for playtest it.

File: code/game/turfs/space/space.dm [hunk 1]
```diff
@@ -13,17 +13,28 @@
...
+			var/image/base_image = image(icon = icon, icon_state = I) //only because byond will not create an image if you do not give it some values
+			base_image.appearance = appearance
+			base_image.icon_state = I
+			var/image/parallax_overlay = image('icons/turf/space_parallax1.dmi', I)
+			parallax_overlay.plane = SPACE_DUST_PLANE
+			parallax_overlay.alpha = 80
+			parallax_overlay.blend_mode = BLEND_ADD
+			base_image.overlays += parallax_overlay
+			parallax_appearances[I] = base_image
+	appearance = parallax_appearances[icon_state]
```
Comment (clusterfack) [inline code/game/turfs/space/space.dm]: You need to either store the above as number indexes by adding 1 to all of them, or access this using ``"[((x + y) ^ ~(x * y) + z) % 25]"``

Comment (DamianX) [reply code/game/turfs/space/space.dm]: yes, on it

**Comment by DamianX** [comment]
9001 hours of testing later this should be good to go

**Pull request merged**